clicommand/artifact_download.go

```diff
@@ -26,7 +26,7 @@ Description:
    'foo/app' will write any matched artifact files to 'foo/app/logs/', relative
    to the current working directory.
 
-   To skip this behaviour, use a <destination> argument with a trailing slash.
+   To avoid this behaviour, use a <destination> argument with a trailing slash.
```

Comment:
This doc tweak was manually applied in https://github.com/buildkite/docs/pull/1063 and this backports it